packages/@okta/courage-for-signin-widget/src/CourageForSigninWidget.js

```diff
@@ -59,6 +59,17 @@ const events = {
 const TextBoxForSigninWidget = TextBox.extend({ events });
 const PasswordBoxForSigninWidget = PasswordBox.extend({ events });
 
+const Form = BaseForm.extend({
+  scrollOnError() {
+    // scrollOnError is true by default. Override to false if `scrollOnError` has been set to false in widget settings.
+    const { settings } = this.options;
+    if (settings.get('features.scrollOnError') === false) {
```

Comment:
as @haishengwu-okta indicated.. it could be only `return option.settings.get('feature.scrollOnError') !== false;` right ?